crates/matrix-sdk/examples/command_bot.rs

```diff
@@ -28,9 +27,7 @@ async fn on_room_message(event: SyncMessageEvent<MessageEventContent>, room: Roo
         };
 
         if msg_body.contains("!party") {
```

Comment:
Don't we have the `body()` method now as well, so we can remove the destructuring here?

```rust
if event.content.body().contains("!party") {
```

Reply:
Yes, weil update this.

Reply:
Wait, that would make it respond to any message, even if it's not a `"msgtype": "text"` (could be a notice by another bot for example). I thought when I read this comment that this would just be a simplification, but it's a change in behavior and I don't get why that would make more sense here than the current behavior.

Reply:
Oh right, we would need to check the msgtype as well, oh well let's leave it as is if you prefer it that way.

Reply:
Thinking about it, there's a much easier improvement to be had. Will include that as a separate commit.